### PR TITLE
Establish ADRs and Document Public Event Calendar Management for OSCAL Team

### LINKED ISSUE #1408: [RFC] Add a Calendar Feed for NIST OSCAL Events to OSCAL Site?
### User Story

As an OSCAL stakeholder and active community member, in order to have a more consistent way to keep abreast of any OSCAL event (dev lunch, model meeting, and mini workshops), I want a hosted calendar to automatically update my email/calendar management client. I also want to know what others think of this, so this is currently a Request for Comment (RFC). :-)

### Goals

- [x] Pick up a technical solution to communicate calendar updates in one consistent, highly available  #1638
- [x] Communicate the use of the updated mechanism and develop a documented process for keeping the event calendar current and up to date #1638
- [x] Implement the change in the website with team support #1642

### Dependencies

- [x] Determination if just using site updates with usnistgov/OSCAL#1407 is sufficient or not, and if this work is prioritized or closed as work we do not plan to do.

### Acceptance Criteria

- [x] Design a solution with common community use cases in mind.
- [x] Implement the calendaring solution, if well-received.
- [x] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [x] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
# Committer Notes

Relates to #1408.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

**Review by Compton-NIST — DISMISSED** [review]

**Comment by aj-stein-nist** [comment]
Hmmm, it seems I dismissed the review and made it stale automatically. Note to self: how to look into reconfiguring that behavior in GH to reduce lag time on all PRs.

File: decisions/0002-record-architecture-decisions.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+1. Manage frequent changes to calendar events efficiently with short notice without waiting upon approval times by team, group, division, or lab leadership
+
+With these high-level requirements in mind, we have several options.
+
+## CMS for the NIST Computer Security Resource Center Website and Calendar
+
+As part of a spike, we evaluated event management [via the content management system that published the csrc.nist.gov website, specifically the Events page](https://csrc.nist.gov/Events). Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 4 at all. The CMS has a permissions system that only works with one group within the respective division for those who manage the content. At this time, the NIST OSCAL Team is composed of staff from teams from two different group's in ITL's Computer Security Division, so requirements 1 and 2 are not easily achievable or unsupported based upon consultation with the web developers that maintain the system. Additionally per requirement 5, there will need to be approval for every change, imposing upon approvers for events where we have historically made frequent changes.
```
Comment (iMichaela) [inline decisions/0002-record-architecture-decisions.md:37]: Should the sentence above read "Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 2 at all." (not 4)?

Comment (aj-stein-nist) [reply decisions/0002-record-architecture-decisions.md:37]:
```suggestion
As part of a spike, we evaluated event management [via the content management system that published the csrc.nist.gov website, specifically the Events page](https://csrc.nist.gov/Events). Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 2 at all. The CMS has a permissions system that only works with one group within the respective division for those who manage the content. At this time, the NIST OSCAL Team is composed of staff from teams from two different group's in ITL's Computer Security Division, so requirements 1 and 2 are not easily achievable or unsupported based upon consultation with the web developers that maintain the system. Additionally per requirement 5, there will need to be approval for every change, imposing upon approvers for events where we have historically made frequent changes.
```

File: decisions/0002-record-architecture-decisions.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Feedback Requested
+
+## Context
+
+The NIST OSCAL Team has necessary touchpoints with the community, among them structured and unstructured meetings with community stakeholders. Many of these meetings are events occuring at set frequencies, while others do not. In all cases, the community, past and present, has expressed the need for invitations to events that are easily accessible with the most current information possible.
+
+## Decision
+
+The NIST OSCAL Team will use [a public shared calendar from a service account in the Office 365 instance for NIST, as documented below](#service-account-with-published-nist-office-365-calendar). The team staff will add, change, and remove event invitations. Community members will view the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files that are a standard that many calendar clients support.
```
Comment (iMichaela) [inline decisions/0002-record-architecture-decisions.md:15]: The sentence "Community members will view the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files that are a standard that many calendar clients support." does not read well. Community members will have access to the .ics calendar invitation files in the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar). These .ics files are standard a standard that many calendar clients support.

File: decisions/0002-record-architecture-decisions.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Consequences
+
+In the past, NIST staff have sent individual calendar invites from their individual accounts' calendars to specific individuals or the oscal-dev@list.nist.gov mailing list at large. Additionally, the team had published individual iCalendar (ICS) files for two of the long-standing public meetings. The latter had not been updated frequent and were out of sync until removed completely with [pull request usnistgov/OSCAL#1614](https://github.com/usnistgov/OSCAL/pull/1614). The former approach, although convenient, presents a few challenges.
```
Comment (wendellpiez) [inline decisions/0002-record-architecture-decisions.md:19]:
```suggestion
In the past, NIST staff have sent individual calendar invites from their individual accounts' calendars to specific individuals or the oscal-dev@list.nist.gov mailing list at large. Additionally, the team had published individual iCalendar (ICS) files for two of the long-standing public meetings. The latter had not been updated frequently and were out of sync until removed completely with [pull request usnistgov/OSCAL#1614](https://github.com/usnistgov/OSCAL/pull/1614). The former approach, although convenient, presents a few challenges.
```

File: decisions/0002-record-architecture-decisions.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# 2. Communicating NIST's OSCAL Events to the Public
```
Comment (nikitawootten-nist) [inline decisions/0002-record-architecture-decisions.md:1]: B: The name of this file is misleading. It should be `0002-communicating-oscal-events.md` or the like.

**Review by nikitawootten-nist — CHANGES_REQUESTED** [review]
The content looks good, but the name of the file is incorrect.

**Review by wendellpiez — APPROVED** [review]
Nice work so far. (Not expecting perfection just yet.)

File: decisions/0002-record-architecture-decisions.md (hunk 4 above)
Comment (aj-stein-nist) [reply decisions/0002-record-architecture-decisions.md:1]: Thanks, I can rename it. Somehow I missed that.

File: decisions/0002-record-architecture-decisions.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Status
+
+Feedback Requested
+
+## Context
+
+The NIST OSCAL Team has necessary touchpoints with the community, among them structured and unstructured meetings with community stakeholders. Many of these meetings are events occuring at set frequencies, while others do not. In all cases, the community, past and present, has expressed the need for invitations to events that are easily accessible with the most current information possible.
+
+## Decision
```
Comment (iMichaela) [inline decisions/0002-record-architecture-decisions.md:13]: I do not think that people (a.k.a. every member of the community) will be willing to open this calendar and merge it into their client. I thought we will use this calendar to invite oscal-dev@list.nist.gov to our event. The advantage of using this calendar would be the fact that any team member would be able to manage the invitations.

**Review by iMichaela — CHANGES_REQUESTED** [review]
Please see the comments suggesting rewording and also the concern regarding the process.

**Review by Compton-NIST — DISMISSED** [review]

File: decisions/0002-record-architecture-decisions.md (hunk 4 above)
Comment (aj-stein-nist) [reply decisions/0002-record-architecture-decisions.md]: Renamed in https://github.com/usnistgov/OSCAL/pull/1638/commits/1f30aeabb6caec56203f3ebf7c80217b190643c2.

File: decisions/0002-communicating-nist-oscal-events-to-the-public.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Feedback Requested
+
+## Context
+
+The NIST OSCAL Team has necessary touchpoints with the community, among them structured and unstructured meetings with community stakeholders. Many of these meetings are events occuring at set frequencies, while others do not. In all cases, the community, past and present, has expressed the need for invitations to events that are easily accessible with the most current information possible.
+
+## Decision
+
+The NIST OSCAL Team will use [a public shared calendar from a service account in the Office 365 instance for NIST, as documented below](#service-account-with-published-nist-office-365-calendar). The team staff will add, change, and remove event invitations. Community members will have access to the .ics calendar invitation files in the web version. These .ics files are [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files - a standard that many calendar clients support.
```
Comment (aj-stein-nist) [inline decisions/0002-communicating-nist-oscal-events-to-the-public.md:15]:
```suggestion
The NIST OSCAL Team will use [a public shared calendar from a service account in the Office 365 instance for NIST, as documented below](#service-account-with-published-nist-office-365-calendar). The team staff will add, change, and remove event invitations. Community members will have access to the .ics calendar invitation files in the web version. These .ics files are [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files - a standard that many calendar clients support.

The NIST OSCAL Team will use the shared calendar to send invitations for public events only to [those subscribed to our mailing lists](https://pages.nist.gov/OSCAL/contact/#oscal-mailing-lists). We will not use it to send invitations to named individuals.
```

File: decisions/0002-record-architecture-decisions.md (hunk 5 above)
Comment (aj-stein-nist) [reply decisions/0002-record-architecture-decisions.md]: I address it in a subsequent change https://github.com/usnistgov/OSCAL/pull/1638/commits/cd973e7a0362402c6f708d313ace8701b139bfbf.

**Review by iMichaela — APPROVED** [review]
LGTM

**Review by nikitawootten-nist — APPROVED** [review]
LGTM

**Review by Compton-NIST — APPROVED** [review]

**Review by JustKuzya — APPROVED** [review]
No objections or corrections on my end - simple edits

**Review by wendellpiez — APPROVED** [review]
Thanks AJ

**Pull request merged**